scripts/benchmarks_ci.py

```diff
@@ -25,6 +25,7 @@
 from glob import iglob
 from itertools import chain
 from logging import getLogger
+from os import path
```

Comment:
nit: Imports from package `os` are not grouped. Already imported the `os` module below, you could use `os.path`.